gitlab/resource_gitlab_environment.go

```diff
@@ -55,18 +58,16 @@ func resourceGitlabEnvironmentCreate(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
-	d.SetId(strconv.Itoa(environment.ID))
+	d.Set("environment_id", environment.ID)
+	d.SetId(fmt.Sprintf("%s/%d", project, environment.ID))
```

Comment:
Can you change this to `%s:%d` so that the ID format matches the other IDs in this provider?